teamcity/README.md

```diff
@@ -149,3 +226,15 @@ Need help? Contact [Datadog support][12].
 [11]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
 [12]: https://docs.datadoghq.com/help/
 [13]: https://www.datadoghq.com/blog/track-performance-impact-of-code-changes-with-teamcity-and-datadog
+[14]: https://github.com/DataDog/integrations-core/blob/master/teamcity/metadata.csv
+[15]: https://github.com/DataDog/integrations-core/blob/master/teamcity/assets/service_checks.json
+[16]: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md
+[17]: https://raw.githubusercontent.com/DataDog/integrations-core/master/teamcity/images/images/authentication.jpg
```

Comment:
These images are not showing up: 
<img width="942" alt="image" src="https://user-images.githubusercontent.com/31313038/204908681-a712c3af-2893-4410-9ff7-d8bcb14996c7.png">

I think it's because the URLs are `.../images/images/authentication.jpg` instead of `../images/authentication.jpg`

Reply:
Good catch, thanks! Fixed!